cmd/logger.go

```diff
@@ -17,78 +17,50 @@
 package cmd
 
 import (
+	"encoding/json"
 	"fmt"
-	"io/ioutil"
+	"os"
 	"path"
+	"path/filepath"
 	"runtime"
+	"strconv"
 	"strings"
+	"time"
 
-	"github.com/Sirupsen/logrus"
 	"github.com/minio/mc/pkg/console"
 	"github.com/minio/minio/pkg/errors"
 )
 
 var log = NewLogger()
 
-// LogTarget - interface for log target.
-type LogTarget interface {
-	Fire(entry *logrus.Entry) error
-	String() string
+type logEntry struct {
+	Level   string   `json:"type"`
+	Message string   `json:"message"`
+	Time    string   `json:"time"`
+	Cause   string   `json:"cause"`
+	Source  string   `json:"source"`
+	Stack   []string `json:"trace"`
```

Comment:
AB wanted to call it `Trace: ` in both console output and in Json format. I chose not to change the structure element name since it only involves the display. If you think this could be a source of confusion, then please let me know, so I can change all references to it as `Trace`.

Reply:
We would need to fix structure field to `Trace` or json name `stack`.  It would confuse in debugging logs with code.

Reply:
Changed `stack` references to `trace`.